Gemfile.lock

```diff
@@ -88,18 +88,18 @@ GEM
     concurrent-ruby (1.2.0)
     cork (0.3.0)
       colored2 (~> 3.1)
-    danger (9.2.0)
+    danger (8.6.1)
```

Comment:
This seems like a downgrade, but I think it's probably ok, I know bumping dunger actually broke some things in Android, not sure if it's also causing any issues for you?

Reply:
Yeah I was confused, but that's what I get when I do `gem install`?

Reply:
Hmm I see the `Gemfile` doesn't specify the version, so it should be using the latest... Maybe you are using a cached version? But that's weird, I don't know how that happened either 😅